app/user/rest.py

```diff
@@ -322,7 +322,7 @@ def create_2fa_code(
     if template.template_type == SMS_TYPE:
         reply_to = template.service.get_default_sms_sender()
     elif template.template_type == EMAIL_TYPE:
-        reply_to = template.service.get_default_reply_to_email_address()
+        reply_to = None
```

Comment:
Nit: This doesn't need to be reassigned here since it's already assigned to None above. This is more explicit, so it's up to you if you'd rather leave it.